Updated the model to address the observed data mismatch during behav sim in regression tests

SrilakshmiDegala opened the pull request:
This model will be further updated once CR-1209458 is fixed

SrilakshmiDegala committed:
Updated the model to address the observed data mismatch during behavioral simulation

This model will be further updated once CR-1209458 is fixed

SrilakshmiDegala committed:
updated settings in the hub

robgraessle commented:
@SrilakshmiDegala The output looks good. Please explain how the reload_tdata_data signal is formatted. It would be good to compare the filter to a Simulink FIR block.

SrilakshmiDegala committed:
Merge branch 'Xilinx:2025.2' into 2025.2_Examples

SrilakshmiDegala committed:
Comparing the filter to a Simulink FIR block.

SrilakshmiDegala committed:
Change FIR filter coefficients in README and added a note

The old design was sending negative coefficients through the reload port while the FIR Compiler GUI was configured for unsigned coefficients. This mismatch in coefficient signedness is one of the root causes of the XSIM vs. hybrid‑sim inconsistency.

SrilakshmiDegala committed:
Fix formatting of note on coefficient signedness

SrilakshmiDegala commented:
1. The old design (single-path) was sending negative coefficients through the reload port while the FIR Compiler GUI was configured for unsigned coefficients. This mismatch in coefficient signedness is one of the root causes of the XSIM vs. hybrid‑sim inconsistency. This requirement applies to both single‑path and multi‑path (e.g., 2‑path/SSR) FIR designs.

2. For multi‑path (e.g., 2‑path/SSR) FIR designs, as explained in CR‑1209458, additional mismatches appeared due to incorrect byte/word ordering of reload-packet can also cause mismatches between hybrid‑sim and XSIM.
